aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -237,6 +237,21 @@ CREATE_UNARY_UFUNC_NO_INPLACE(signbit);
 CREATE_UNARY_UFUNC_FUNCTIONAL(isnan);
 CREATE_UNARY_UFUNC_FUNCTIONAL(isinf);
 
+// pow function for Sparse CSR, since square is composite - this will also enable square for Sparse CSR
+Tensor& pow_sparse_csr_scalar_out(const Tensor& self, const Scalar& exponent, Tensor& result) {
+  return unary_op_out<Tensor& (*)(const Tensor&, const Scalar&, Tensor&)>(&at::pow_outf, self, result, exponent);
```

Comment:
Just curious, do you really need to specify the type? Isn't
```suggestion
  return unary_op_out(&at::pow_outf, self, result, exponent);
```
enough? Same in the next function.

Reply:
I had tried that, but the build failed with the following error:

```cpp
/home/krshrimali/pytorch/aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp: In function 'at::Tensor& at::native::pow_sparse_csr_scalar_out(const at::Tensor&, const c10::Scalar&, at::Tensor&)':
/home/krshrimali/pytorch/aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp:243:60: error: no matching function for call to 'unary_op_out(<unresolved overloaded function type>, const at::Tensor&, at::Tensor&, const c10::Scalar&)'
  243 |   return unary_op_out(&at::pow_outf, self, result, exponent);
      |                                                            ^
/home/krshrimali/pytorch/aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp:75:9: note: candidate: 'template<class F, class ... Args> at::Tensor& at::{anonymous}::unary_op_out(F, const at::Tensor&, at::Tensor&, Args ...)'
   75 | Tensor& unary_op_out(F op_out, const Tensor& self, Tensor& result, Args... args) {
      |         ^~~~~~~~~~~~
/home/krshrimali/pytorch/aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp:75:9: note:   template argument deduction/substitution failed:
/home/krshrimali/pytorch/aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp:243:60: note:   couldn't deduce template parameter 'F'
  243 |   return unary_op_out(&at::pow_outf, self, result, exponent);
```

The `unresolved overloaded function type` in the error made me specify the type explicitly.

Reply:
I hadn't realised it's an overloaded function, thanks!